Feat: comment 엔티티 구현

Shsin9797 opened the pull request:
## 🔎 작업 내용

> Comment entity 기능 1차 구현


## 💬 리뷰 요구사항 (선택)

>지금은 memberId, post_id,  String content 3개를 넣을수있는 생성자를 추가로 만들었는데, 이거 세개외에 또 꼭 필요한거 있나요? 


## 📸 이미지 첨부 (선택)



## ➕ 이슈 링크

- [Feature/comment #7] https://github.com/kakao-tech-campus-2nd-step3/Team29_BE/issues/7

yugyeom-ghim committed:
Init: 프로젝트 기본설정 세팅

- 프로젝트 생성
- .gitignore설정
- 프로젝트 의존성 추가
- application.yml 설정파일 구성

yugyeom-ghim committed:
Init: 프로젝트 기본 구조 및 공통 컴포넌트 설정

- 공통 설정 클래스 추가 (JPA, QueryDSL, Swagger, Web)
- 공통 도메인 엔티티 (RootEntity) 정의
- 예외 처리 관련 클래스 및 타입 구현
- JSON 변환을 위한 AttributeConverter 추가
- 유틸리티 클래스 (Math) 추가

yugyeom-ghim committed:
Merge branch 'kakao-tech-campus-2nd-step3:Develop' into Develop

Shsin9797 committed:
Chore: Set Post folder structure

Shsin9797 committed:
Feat: Set Post domain entity

Shsin9797 committed:
Feat: Set PostRepository as JpaRepository interface

Shsin9797 committed:
Chore: Set Comment Folder Structure

Shsin9797 committed:
Feat: Set Comment entity member variable

Shsin9797 committed:
Feat: Set Comment JpaRepository

Shsin9797 committed:
Fix: Edit Comment entity

Shsin9797 committed:
Fix: Edit Post entity

Shsin9797 committed:
Style: reformat Comment entity

yunjunghun0116 reviewed (approved):
고생하셨습니다 :)

yugyeom-ghim reviewed (changes requested):
여기도 Post에 리뷰달았던 내용들 수정해주시면 Approve 하겠습니다!

Shsin9797 committed:
Fix : [코드 리뷰 반영] Comment Entity
- root entity 공통 코드 삭제 (생성시각, 수정시각)
- Repository 타입 BigInteger -> Long으로 변경
- 클래스 멤버변수명 Snake -> Camel 스타일로 변경
- 컬럼 length 지정
